### DISCUSSION TITLE
4x8 llama 3 crash

### DISCUSSION THREAD

**Discussion by nnyanpasu** [YellowRoseCx/koboldcpp-rocm#43] [General]
well normal 8b runs fine on my 7900xtx, but Chaotic-Soliloquy-4x8B.Q4_K_M is crashing// ver KoboldCPP-v1.63.yr1-ROCm
![image](https://github.com/YellowRoseCx/koboldcpp-rocm/assets/136471054/4dcf2611-d93a-47c7-aa85-0d8cf53b5cd2)
gonna try other 4x8b

**Comment by Dajinu** [comment]
Got exactly the same error.

**Comment by YellowRoseCx** [comment]
I'll have to check it out on Windows sometime soon, but I know it's not giving me that issue on Linux with this command:
`python koboldcpp.py --threads 6 --blasthreads 6 --usecublas --gpulayers 25 --blasbatchsize 512 --contextsize 4096 --model /media/yellowrose/AI/ChaoticSoliloquy-4x8B-Q4_K_S.gguf`

**Reply by waynekwal** [reply]
I replicated those settings in the most current Windows release and it's still crashing with an access violation.
